src/app/app.component.html

```diff
@@ -17,7 +17,8 @@ <h1>JS Band Store</h1>
       <li><a routerLink="cart" routerLinkActive="active">Cart</a></li>
     </ul>
   </nav>
-</div>
+  <app-catalog [authToken]="token"></app-catalog>
```

Comment:
why do you need to pass token as prop? you should store token inside auth.service and use it where you need it via DI